iothub_client/tests/iothubclient_openssl_engine_e2e/Testing/Temporary/CTestCostData.txt

```diff
@@ -0,0 +1 @@
+---
```

Comment:
this supposed to be included here?

Reply:
Wow: no. How did that happen - we might be lacking some .gitignore statements. I'll add one to ignore /Testing created by ctest.